lib/conjur/fpm/package.sh

```diff
@@ -63,16 +67,26 @@ mkdir -p opt/conjur/etc
 
 [ -d opt/conjur/"$project_name"/distrib ] && mv opt/conjur/"$project_name"/distrib /
 
-fpm -s dir -t deb -n conjur-$project_name -v $version -C . \
-	--maintainer "Conjur Inc." \
-	--vendor "Conjur Inc." \
+for file_type in deb rpm
```

Comment:
why does this appear twice? line 70 and again in 33?

Reply:
once for the dev package and one for the prod. this logic was here before my change.